packages/p2p/src/components/my-ads/buy-ad-payment-methods-list.scss

```diff
@@ -0,0 +1,5 @@
+.buy-ad-payment-methods-list {
```

Comment:
Added `max-width` property to constraint the dropdown list width:
### Before
<img width="776" alt="Screenshot 2022-06-10 at 3 42 28 PM" src="https://user-images.githubusercontent.com/103016120/173020860-43952af6-abbf-4b8e-b895-b1a86adce5eb.png">

### After
<img width="668" alt="Screenshot 2022-06-10 at 3 43 10 PM" src="https://user-images.githubusercontent.com/103016120/173020886-06fc6585-2b1b-468f-92eb-279644c9b33d.png">